bzt/modules/jmeter.py

```diff
@@ -260,19 +261,59 @@ def __port_is_free(self, port_num):
             return False
 
     @staticmethod
-    def __apply_ramp_up(jmx, ramp_up):
+    def __apply_ramp_up(jmx, load):
         """
-        Apply ramp up period in seconds to ThreadGroup.ramp_time
+        Apply ramp up period in seconds to ThreadGroups
         :param jmx: JMX
         :param ramp_up: int ramp_up period
         :return:
         """
+        ramp_up = int(load.ramp_up)
+
+
+
+        for group in jmx.enabled_thread_groups():
+            if group.tag == 'ThreadGroup':
```

Comment:
We should also support Ultimate Thread Groups, taking max threads value from it.